test/Microsoft.Azure.Devices.Edge.Test/Module.cs

```diff
@@ -12,20 +12,24 @@ namespace Microsoft.Azure.Devices.Edge.Test
 
     public class Module : ManualProvisioningFixture
     {
-        private const string DefaultSensorImage = "mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0";
+        private string GetTempSensorImage()
+        {
+            const string DefaultSensorImage = "mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0";
+            return Context.Current.TempSensorImage.GetOrElse(DefaultSensorImage);
+        }
 
+        [Category("TempSensor")]
```

Comment:
Why add this category?

Reply:
I get that you might want to run all the tempSensor-related tests together, but I think you could do that with `--filter Name~Temp` (or `--filter FullyQualifiedName~Microsoft.Azure.Devices.Edge.Test.Module.Temp`).